test/verify/check-services

```diff
@@ -481,5 +481,63 @@ WantedBy=default.target
 
         self.allow_restart_journal_messages()
 
+    def testConditions(self):
```

Comment:
Looking good, I think we need to skip this test on rhel-7.4 because we only test what will actually be delivered on 7.4 there and this won't be, at least until some asks for it to be backported.